packages/common/src/hooks/useListData/useListData.test.tsx

```diff
@@ -102,7 +102,7 @@ describe('useListData', () => {
   it('calls error boundary on server error', async () => {
     const ErrorTest = () => {
       const { data } = useListData(
-        { key: 'message' },
+        { initialSortBy: { key: 'message' } },
```

Comment:
just changing the calling signature to be an object `{initialSortBy, initialFilterBy}` rather than just an `initialSortBy`